package-lock.json

```diff
@@ -5348,7 +5348,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```

Comment:
These changes to `package-lock.json` aren't required for this change, can we revert them?

Reply:
Oh, my mistake, I've reverted it